src/quantum/index.jsx

```diff
@@ -30,20 +29,20 @@ export default class QuantumIndex extends React.Component {
       match: { params },
     } = this.props;
     const urlParams = URL2Object(location.search);
-    const bits = urlParams.bits || params.bits;
+    const bits = urlParams.bits || Number.parseInt(params.bits, 10);
```

Comment:
I know it's very tempting but in the future try not to have drive-by fixes to make the PRs extra clear.
If it helps `git add --patch` has done wonders to me to separate changes from the same file.

Reply:
I noticed this problem, It thought it was just something that had to be fixed to get the pages working.  Turns out that this is an existing problem: https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/348